cypress/e2e/home.cy.js

```diff
@@ -9,7 +9,7 @@ describe('Home screen', function () {
   })
 
   it('allows the user to edit an existing form', function () {
-    cy.contains('Edit').click()
+    cy.contains('edit').click()
```

Comment:
I think we probably want to keep the Edit / View in Runner links capitalised but can check with Hannah if we're not sure

Reply:
Good suggestion about the copy. I had a chat with Hannah and I've reverted those changes.